src/main/resources/application.yml

```diff
@@ -1,7 +1,9 @@
 dancier:
-  kikriki:
+  kikeriki:
     topic: test
 management:
+  server:
+    port: 8081
```

Comment:
We in general have two ports in services:
one that is being exposed in the internet (via traefik) and one that is not (for monitoring and so on)